definition_of_ready.md

```diff
@@ -0,0 +1,31 @@
+# Definition of ready (DoR)
+- Each user story should fulfill the INVEST criteria (see e.g. [Wikipedia](https://en.wikipedia.org/wiki/INVEST_(mnemonic)), [srum-in-der-praxis.de](https://scrum-in-der-praxis.de/glossary/invest))
+  - See https://www.knowledgehut.com/tutorials/scrum-tutorial/definition-of-ready
+    - Independent (of all others)
+    - Negotiable (not a contract for a specific feature)
+    - Valuable (a vertical slice)
+    - Estimable (to a decent approximation)
+    - Small (enough to fit in a single iteration)
+    - Testable (even if the test doesn't exist yet)
+
+## In detail
+- The user story is Well-defined:
+  - The user story should be written in the user story format, see for example [Wikipedia](https://en.wikipedia.org/wiki/User_story#Principle).
```

Comment:
```suggestion
  - The user story should be written in the user story format, see for example [Wikipedia](https://en.wikipedia.org/wiki/User_story#Principle).
  - e.g. `As a <role> I can <capability>, so that <receive benefit>`
```